src/tactic/ui/tools/ingest_wdg.py

```diff
@@ -1824,6 +1834,9 @@ def execute(my):
                 process = "publish"
 
 
+            if update_sobject_found:
```

Comment:
actually you need to check if update_proccess is not empty as well
if update_process and update_sobject_found: